routers/web/web.go

```diff
@@ -104,7 +104,7 @@ func ctxDataSet(args ...any) func(ctx *context.Context) {
 }
 
 // Routes returns all web routes
-func Routes() *web.Route {
+func Routes(middlewares ...any) *web.Route {
```

Comment:
`middlewares ...any` introduces strange smell to the code base, no comment, and it might be abused in the future.